tests/codeception/acceptance/100-BackendAdmin/BackendAdminCest.php

```diff
@@ -569,10 +569,8 @@ public function logoutAdminUserTest(\AcceptanceTester $I)
         $I->see('Dashboard');
         $I->click('Logout');
 
-        // Removed as we now unset the session cookie at logout
-        //$I->see('You have been logged out');
-
         $I->amOnPage('/bolt');
-        $I->see('Please log on');
+
+        $I->seeInCurrentUrl('/bolt/login');
```

Comment:
Is switching to BK killing flashes?

This, and a few of the other checks for flashes, were to catch a recurring problem where a change in the code base "somewhere" would stop some messages being displayed, or in some way interfered with, and get missed during regular "monkey tests".